packages/now-cli/src/commands/dns/add.ts

```diff
@@ -117,3 +133,63 @@ export default async function add(
 
   return 0;
 }
+
+function readConfirmation(
```

Comment:
We already have a function for this. See `src/util/prompt-bool.ts`.

Reply:
I will check that. Thanks!